rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,174 @@
+
+## Author(s)
+*DCP PM*
+
+**Note:** *This proposal is based on ideas from the **HCA DCP Technical Decision-Making** white paper written by Bruce Martin with contributions from Tony Burdett, Laura Clarke, Brian O’Connor, Brian Raymor, and Tim Tickle.*
+
+## Shepherd
+[Brian Raymor](brianraymor@chanzuckerberg.com)
+
+## Motivation
+
+There are a number of limitations with our current process which need to be addressed:
+
+- For community members who want to propose enhancements to either technical software or governance, it is often unclear where to start - *for example, which HCA slack channel or project member to approach with a proposal.*
+
+  - There is no common template for design proposal submissions.
+
+  - There is no common process describing how the community reviews and approves submitted proposals.
+
+  - There is no formal assistance to guide a contributor through the RFC process. 
+
+- New community members experience difficulties when _onboarding_. There is no curated repository of up-to-date design documents to assist with learning about the software. 
+
+- To ensure consistency of the architectural design, senior technical leaders must monitor multiple work streams or hope that they are notified of potential technical changes - *this is a significant concern for cross-cutting issues such as security or the data model*.
+
+To address these needs, the DCP RFC process defines a transparent and standard method for community members to propose and build consensus around substantial enhancements to DCP software or governance.
+
+### When is a RFC required?
+
+Any proposal that would benefit from additional review or design before being implemented is a good candidate for a RFC.
+
+- A RFC is required if the implemented design would be described in written communication to the community.
+- A RFC is required if a design impacts the scope of multiple projects.
+- A RFC is required if a technical initiative (refactoring, major architectural change) impacts the community.
+- A RFC is required for all changes to DCP governance.  
+
+### Revising RFC(s)
+
+How are RFC(s) maintained as *living documents* in an iterative development process where the original design may substantially change based on implementation experiences?
+
+Where necessary, existing RFCs can be revised using the same process (a proposed change submitted as a pull request and subject to community review and approval). **Major changes should result in a new RFC.**
+
+### What are the roles in the RFC process?
+
+- **Authors** are DCP community members.
+
+- **Shepherds** are members of *DCP PM* assigned to assist the **Author(s)** and ensure that a RFC progresses to closure.
+
+- **Approvers** for governance RFC(s) are the Project Leads and Product Owners from *DCP PM*.
+
+- **Approvers** for software RFC(s) are the Technical Leads from *DCP Architecture*. 
+
+- **Reviewers** are DCP community members.
```

Comment:
Suggest clarifying that the DCP community includes both implementors and users of the DCP.  In other words, RFCs affecting third-parties not affiliated with DCP implementation should also be welcomed as reviewers.

Reply:
WFM.